admin/app/components/solidus_admin/ui/toast/component.rb

```diff
@@ -3,16 +3,19 @@
 class SolidusAdmin::UI::Toast::Component < SolidusAdmin::BaseComponent
   SCHEMES = {
     default: %w[
-      bg-gray-800 text-white
+      bg-black text-white
```

Comment:
@rainerdema the class `xxx-black` corresponds to `rgb(34, 34, 34)`, HEX color `#222222` which is the color used for the primary button, for example. I've added the full black `#000000` to fill the toast background. I think is worth adding, probably we'll need it in the future when we'll develop the dark/light theme mode.
We could call it `full-black` or `pure-black`, and it can stay under the [Secondary palette](https://github.com/solidusio/solidus/blob/main/admin/config/solidus_admin/tailwind.config.js.erb#L28).

Let me know your thoughts on it!